doc/doxygen/scripts/filter

```diff
@@ -75,7 +75,28 @@ while (<>)
     # "file :", which doxygen doesn't recognize:
     s#file:[ \t]*$#file :#g;
 
-
+    # handle tutorial DOIs of the form @dealiiDOI{link,imglink}
+    # and @dealiiDOI{link}
+    if (m/(\@dealiiDOI\{([^\}]+)\})/)
+    {
+	@args = split(',', $2, 2);
+	$url = @args[0];
+	$imgurl = @args[1] || '';
+	$text = "\@note The initial version of this tutorial can be cited as ";
```

Comment:
I suppose that by using the word "tutorial" this shortcut cannot be reused in the code-gallery examples. Should we perhaps replace "tutorial" by a more generic term, such as "work"?